k8s/src/main/scala/io/buoyant/k8s/EndpointsNamer.scala

```diff
@@ -239,13 +235,46 @@ object EndpointsNamer {
   protected type PortMap = Map[String, Int]
   protected type NumberedPortMap = Map[Int, String]
 
-  private case class Endpoint(ip: InetAddress, nodeName: Option[String])
+  private[EndpointsNamer] case class Endpoint(ip: InetAddress, nodeName: Option[String])
 
-  private object Endpoint {
+  private[EndpointsNamer] object Endpoint {
     def apply(addr: v1.EndpointAddress): Endpoint =
       Endpoint(InetAddress.getByName(addr.ip), addr.nodeName)
   }
 
+  private[EndpointsNamer] implicit class RichSubsetsSeq(
```

Comment:
any change here or just a move?

Reply:
an (unnecessary) move for changes I didn't end up making. Didn't mean to commit this.